reana/reana_dev/utils.py

```diff
@@ -335,7 +335,7 @@ def run_command(cmd, component="", display=True, return_output=False):
         sys.exit(err.returncode)
 
 
-def display_message(msg, component=""):
+def display_message(msg, component="", color="white"):
```

Comment:
Is this change necessary in your setup? It may not work well with certain colour schemes to force white colour... It would be better not to touch `display_message()` since it is an unrelated change to the issue at hand.